ci-operator/step-registry/gcp/provision/xpn-byo-hosted-zone-min-perm-passthrough-serviceaccount/gcp-provision-xpn-byo-hosted-zone-min-perm-passthrough-serviceaccount-commands.sh

```diff
@@ -12,5 +12,12 @@ set -o pipefail
 EXIT_CODE=100
 trap 'if [[ "$?" == 0 ]]; then EXIT_CODE=0; fi; echo "${EXIT_CODE}" > "${SHARED_DIR}/install-pre-config-status.txt"' EXIT TERM
 
-echo "$(date -u --rfc-3339=seconds) - Enabling the IAM service account of minimal permissions, i.e. no permissions creating/deleting firewall-rules and binding a private zone to the shared VPC in the host project, for deploying OCP cluster into GCP shared VPC..."
-cp "${CLUSTER_PROFILE_DIR}/ipi-xpn-no-fw-no-dns-sa.json" "${SHARED_DIR}/xpn_byo-hosted-zone_min_perm_passthrough.json"
+if [[ -z "${PRIVATE_ZONE_PROJECT}" ]] || [[ "${PRE_CREATE_PRIVATE_ZONE}" == "yes" ]]; then
```

Comment:
```suggestion
if [[ "${PRE_CREATE_PRIVATE_ZONE}" == "yes" ]]; then
```

I think we can remove `PRIVATE_ZONE_PROJECT` condition here. Whatever byo-private-hosted-zone in service project or byo-private-hosted-zone in the 3rd project, we always need to enable `PRE_CREATE_PRIVATE_ZONE`